cypress/integration/rules_spec.js

```diff
@@ -106,6 +106,35 @@ describe('Rules page', () => {
         });
     });
 
+    it('editing a rule and returning to the rules page should show the new changes', () => {
+      cy.visit('/#/rules');
+      cy.contains('table tbody tr a', testRuleName)
+      .and('have.attr', 'href', `#/rules/rule/${testRuleName}`).click();
```

Comment:
Can we update the indentation here for readability?

```suggestion
cy.contains('table tbody tr a', testRuleName)
  .and('have.attr', 'href', `#/rules/rule/${testRuleName}`)
  .click();
```

Reply:
changed